engine/client/s_load.c

```diff
@@ -158,7 +158,7 @@ wavdata_t *S_LoadSound( sfx_t *sfx )
 		Sound_Process( &sc, SOUND_11k, sc->width, sc->channels, SOUND_RESAMPLE );
 	else if( sc->rate > SOUND_11k && sc->rate < SOUND_22k ) // some bad sounds
 		Sound_Process( &sc, SOUND_22k, sc->width, sc->channels, SOUND_RESAMPLE );
-	else if( sc->rate > SOUND_22k && sc->rate < SOUND_44k ) // some bad sounds
+	else if( sc->rate > SOUND_22k && sc->rate < SOUND_44k || sc->rate > SOUND_44k ) // some bad sounds
```

Comment:
`sc->rate != SOUND_44k`?

Reply:
No, that's wrong. This will cause 11k and 22k sounds to be resampled to 44k for no reason.

Reply:
Sorry, what I meant is:
```c
else if( sc->rate > SOUND_22k && sc->rate != SOUND_44k ) ...
```

Reply:
Ok, good. Completed.